### ISSUE TITLE
What Are The Pros/Cons vs JimFS?

### ISSUE THREAD

**Issue by opticyclic** [marschall/memoryfilesystem#72]
With no other knowledge, [JimFs](https://github.com/google/jimfs) seems to be the choice to make for an im-memory filesystem for testing purely because of the Google backing.

However, your README gives me a lot of confidence that memoryfilesystem could be a good choice too.

Can you update the README/FAQ with some pros/cons of your project vs JimFS?
e.g. how would I convince other members of a team to use your project if they have never heard of it but have heard of Google?

**Comment by marschall** [comment]
Yes, to be honest I'm not too familiar with JimFs, when I started the project I was not aware of JimFs.

May I ask what your use case and specific requirements are?
It should be pretty easy for you to check both out and see which one works better for you. In general the code under test and the test code should not have any dependencies outside of `java.nio.file`. This code should get a `FileSystem` or `Path` injected via IoC. The only thing that should be file system specific should be a small piece of set up code. That can either be Spring/Guice/Dagger/.. or a [JUnit Rule](https://github.com/marschall/memoryfilesystem#next-steps).
